code/modules/mob/mob_defines.dm

```diff
@@ -193,7 +193,7 @@
 	var/move_on_shuttle = 1 // Can move on the shuttle.
 	var/captured = 0 //Functionally, should give the same effect as being buckled into a chair when true.
 
-	var/movement_speed_modifier = 1
+	var/movement_speed_modifier = 6
```

Comment:
It needed to be increased to allow for speed overhead, it was required for the hyperzine instigated speed boost to work as intended. Actual speed is altered elsewhere, specifically run and walk speed.

The changes are all very simple and deploying it locally shouldn't be an issue.
I invite you to investigate for yourself.